### PR TITLE
Milestone 2: Portfolio finish mobile version

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
## In this Milestone my Learning objectives :

- Perfect my HTML semantics best practices knowledge
- Perfect my CSS3 layouts using Flexbox to place elements on the page.
- Perfect my knowledge of GitHub flow
- Perfect my professional documentation knowledge
- Was able to understand how to plan my HTML to come up with the desired markup
- Understand how to parse a Figma design to create a UI.
- Use CSS Grid to place elements on the page.
- Build a personal portfolio mobile site.

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @Chandan-devs-tech,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

- Kindly use the new [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the sections exist in yours and placeholders are replaced with your project and personal data. This is crucial to have a professional and instructive repository. 📝👨‍💼

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -42,32 +53,228 @@ <h1>
...
           </div>
         </section>
       </main>
+      <!-- Work Text Section -->
+      <section>
+        <div>
+          <p class="work-text">
+            My Recent<br />
+            Works
+          </p>
```
Comment (Lizdev-05) [inline index.html:106]: Great job so far 😎. Kindly change the `p` tag to the `h` tag. This is a title and heading tags indicate headings on a webpage by using code to tell a web browser how to display content. `h` tags help improve accessibility for people who can't easily read screens.🟢☺️

File: index.html [hunk 2]
```diff
@@ -42,32 +53,228 @@ <h1>
...
+      <!-- Language Section -->
+      <section class="language-main-container">
+        <div class="language-sub-container">
+          <img
+            src="Assets/icon-languages.svg"
+            alt="languages"
+            width="130px"
+            height="130px"
+          />
+          <p class="language-container-text">Languages</p>
```
Comment (Lizdev-05) [inline index.html:230]: According to the requirement, you are to make use of a `ul` tag for the technologies categories, kindly change the `p` tag to make use of a `ul` tag 😊😎

File: index.html [hunk 3]
```diff
@@ -42,32 +53,228 @@ <h1>
...
+          </ul>
+        </div>
+        <div class="language-sub-container">
+          <img
+            src="Assets/icon-frameworks.svg"
+            alt="frameworks"
+            width="130px"
+            height="130px"
+          />
+          <p class="language-container-text">Frameworks</p>
```
Comment (Lizdev-05) [inline index.html:245]: According to the requirement, you are to make use of a `ul` tag for the technologies categories, kindly change the `p` tag to make use of a `ul` tag 😊😎

File: index.html [hunk 4]
```diff
@@ -42,32 +53,228 @@ <h1>
...
+          </ul>
+        </div>
+        <div class="language-sub-container">
+          <img
+            src="Assets/icon-skills.svg"
+            alt="skills"
+            width="130px"
+            height="130px"
+          />
+          <p class="language-container-text">Skills</p>
```
Comment (Lizdev-05) [inline index.html:261]: According to the requirement, you are to make use of a `ul` tag for the technologies categories, kindly change the `p` tag to make use of a `ul` tag 😊😎

Comment (Lizdev-05) [inline index.html:261]: According to the requirement, you are to make use of a `ul` tag for the technologies categories, kindly change the `p` tag to make use of a `ul` tag 😊😎

File: index.html (hunk 3 above)
Comment (Lizdev-05) [inline index.html:245]: According to the requirement, you are to make use of a `ul` tag for the technologies categories, kindly change the `p` tag to make use of a `ul` tag 😊😎

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Chandan-devs-tech, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct GitHub Flow 👍

- Good commit history 💰

- Good PR title/description 💯

- All linters passed 🏅

- Personalised README file 👍
- Great design ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Chandan-devs-tech) [reply index.html:230]: In place of p tag shall I use div to it

**Review by Gambit142 — APPROVED** [review]
Hi @Chandan-devs-tech 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status Approved 🟢 

### Highlight
- All linters checks passed ✔️
- Descriptive pull request summary and title ✔️ 
- Professional README.md file ✔️ 

### Optional Changes

- [ ] For Future projects, kindly make your commit messages in imperative tense and capitalized.

#### Bad Commit Message
> `Updated` some SVG files in the assets folder

#### Good Commit Messages
>` Update` some SVG files in the assets folder

- Kindly consider the comments tagged `OPTIONAL` in the comments under the review. 👍 💻 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**